spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -306,6 +327,13 @@ def close_all_figures(self):
         """Close all the figures in the thumbnail scrollbar."""
         self.thumbnails_sb.remove_all_thumbnails()
 
+    def keyPressEvent(self, event):
+        """Define Shortcut."""
+        if (event.key() == Qt.Key_C) and (
+                event.modifiers() & Qt.ControlModifier):
```

Comment:
Can you avoid hardcoding the Copy shortcut to Ctrl-C? It is configurable in our preferences to be anything the user wishes, so presumably it should use the shortcut defined there.

Reply:
I'll check how to implement the shortcut in spyder and apply it.

Reply:
:+1: